foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -52,6 +52,17 @@
             ]
           }
         ]
+      },
+      {
+        "name": "hive-universal-config-yaml-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_11_0.HiveUniversalConfigYAMLMigration",
+        "fileSets": [
+          {
+            "includes": [
+              "*-deploy/src/main/resources/apps/spark-infrastructure/values.yaml"
+            ]
+          }
+        ]
```

Comment:
Should include this migration in the Automatic Upgrade section of the draft release notes

Reply:
updated!